lib/collection/src/collection/resharding.rs

```diff
@@ -6,11 +7,16 @@ use crate::shards::shard::{PeerId, ShardId};
 pub struct State {
```

Comment:
Let's maybe call it ReshardingState. Navigation and refactoring will be easier

Reply:
Just use `resharding::State`.... (but OK 🥲)

Reply:
Done.